internal/providers/terraform/azure/app_service_plan.go

```diff
@@ -22,6 +22,9 @@ func NewAzureRMAppServicePlan(d *schema.ResourceData, u *schema.UsageData) *sche
 	skuRefactor := ""
 	os := "windows"
 	capacity := d.Get("sku.0.capacity").Int()
+	if capacity == 0 {
+		capacity = 1
+	}
```

Comment:
There was a new helper added with the HCL stuff so I think we can do:
```suggestion
	capacity := d.GetInt64OrDefault("sku.0.capacity", 1)
```

Reply:
This is cool! Thanks! :)